### PR TITLE
migrated kmlstring to sbcd-common

### PR DESCRIPTION
**Pull request by pjacobsam**

### CODE REVIEW

File: src/components/controls/view/km-lstring-value.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div :id="$attrs.id" class="scbd-controls km-form-group form-group">
```
Comment (blaisefonseca) [inline src/components/controls/view/km-lstring-value.vue]: what are these css classes `km-formp-group form-group`
why is there no km-lstring-value

File: src/utils/filters.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    if(!entry.en) entry.en = entry.ar;
+    if(!entry.en) entry.en = entry.zh;
+  
+    if(!entry.fr) entry.fr = entry.en;
+    if(!entry.es) entry.es = entry.en;
+    if(!entry.ru) entry.ru = entry.en;
+    if(!entry.ar) entry.ar = entry.en;
+    if(!entry.zh) entry.zh = entry.en;
+    return entry;
+  }
```
Comment (blaisefonseca) [inline src/utils/filters.js]: can you move this to `services/util/lstring.js`

File: src/components/controls/view/km-lstring-value.vue [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div :id="$attrs.id" class="scbd-controls km-lstring-value">
+    <div :class="`input-group input-lang-${locale}`">
+      <div v-if="type === 'string'" class="form-control km-value km-value-ml-string"
```
Comment (blaisefonseca) [inline src/components/controls/view/km-lstring-value.vue]:
```suggestion
      <div v-if="type === 'string'" class="form-control km-value km-value-string" 
```

File: src/components/controls/view/km-lstring-value.vue [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div :id="$attrs.id" class="scbd-controls km-lstring-value">
+    <div :class="`input-group input-lang-${locale}`">
+      <div v-if="type === 'string'" class="form-control km-value km-value-ml-string" 
+           :dir="direction(valueLstring, locale)">
+            {{ valueLstring }}
+      </div>
+      <div v-else-if="type === 'html'" class="form-control km-value km-value-ml-html ck-content"
```
Comment (blaisefonseca) [inline src/components/controls/view/km-lstring-value.vue]:
```suggestion
      <div v-else-if="type === 'html'" class="form-control km-value km-value-lstring-html ck-content" 
```

File: src/utils/filters.js (hunk 2 above)
Comment (pjacobsam) [reply src/utils/filters.js]: moved utils from root to services and created lstring file

File: src/components/controls/view/km-lstring-value.vue (hunk 1 above)
Comment (pjacobsam) [reply src/components/controls/view/km-lstring-value.vue]: replaced km-form-group with km-lstring-value class

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**